free-programming-playgrounds.md

```diff
@@ -80,6 +80,7 @@
 ### Flutter
 
 * [Codepen](https://codepen.io/topic/flutter/templates)
+* [Flutter Studio](https://flutterstudio.app/)
```

Comment:
In reference to [CONTRIBUTING](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md), remove the trailing slash and the Travis CI build should pass

> on root domains, strip the trailing slash: http://example.com instead of http://example.com/

Reply:
Yep, did that. Thank you !!

Reply:
@Harshit-lab anytime!